http-source/src/main/resources/application.yml

```diff
@@ -0,0 +1,8 @@
+server:
+  port: 0
+name: ${spring.application.name:http-source}
```

Comment:
Revisiting @ilayaperumalg 's comment about the pom having a space here. Now I know I'm not 100% crazy. Having this file declare a `name` key and also having maven tokenization kick in is *REALLY* misleading IMO.
I'll submit a PR to change maven's to `@`